protected/views/workspace/share.php

```diff
@@ -27,26 +28,29 @@
 
 
 ?>
-<div class="col-xs-12">
-    <?php
-    $form = ActiveForm::begin([
-        'method' => 'POST',
-        "type" => ActiveForm::TYPE_HORIZONTAL,
-        'formConfig' => [
-            'showLabels' => true,
-            'defaultPlaceholder' => false
-        ]
-    ]);
-
-    echo $model->renderForm($form);
-    ?>
-    <div class="col-xs-offset-11"><button type="submit" class="btn btn-primary">Share</button></div>
-    <?php
-    $form->end();
-    ?>
-    <h2><?=\Yii::t('app', 'Already shared with')?></h2>
-    <?php
-    echo $model->renderTable();
-    ?>
+<div class="col-xs-12 share-form">
+    <div class="col-xs-12 permissions-form">
+        <?php
+        echo Html::tag('h2', \Yii::t('app', 'Users and permissions'));
+        $form = ActiveForm::begin([
+            'method' => 'POST',
+            "type" => ActiveForm::TYPE_HORIZONTAL,
+            'formConfig' => [
+                'showLabels' => true,
+                'defaultPlaceholder' => false
+            ]
+        ]);
+
+        echo $model->renderForm($form);
+        $form->end();
+        ?>
+        <button type="submit" class="btn btn-primary">Share</button>
+    </div>
+    <div class="col-xs-12 list-shared">
```

Comment:
There is no need for this wrapping element here, it should be part of the model (somewhere in the `->renderTable()`) function.
Since we're repeating this on every share page.